### PR TITLE
Remove CLIENT_TYPE_WEB_SERVER

### PR DESCRIPTION
**Pull request by ekzhang**
This is here for legacy internal reasons. See the SecretMake PR on the monorepo.

### CODE REVIEW

File: modal_proto/api.proto [hunk 1]
```diff
@@ -36,7 +36,7 @@ enum ClientType {
   CLIENT_TYPE_WORKER = 2; // deprecated
   CLIENT_TYPE_CONTAINER = 3;
   CLIENT_TYPE_SERVER = 4; // deprecated
-  CLIENT_TYPE_WEB_SERVER = 5;
+  CLIENT_TYPE_WEB_SERVER = 5; // deprecated
```
Comment (erikbern) [inline modal_proto/api.proto:39]: i think we can maybe use `[deprecated=true]` for these

Comment (ekzhang) [reply modal_proto/api.proto]: Nice I didn't realize that works!

**Comment by luiscape** [comment]
`CLIENT_TYPE_WEB_SERVER` is used in audit logs to indicate that a request originated from the web UI (cc @irfansharif). Are we replacing this with something else?

image

**Comment by ekzhang** [comment]
Ah I thought it was just used for `service_client()`. That's my bad then.

**Comment by ekzhang** [comment]
Note that the IP address that we're listing is an AWS IP address though, it just comes from the EC2 address of modal-web.

**Pull request merged**